ui/hooks/gasFeeInput/useGasFeeInputs.js

```diff
@@ -114,12 +117,10 @@ export function useGasFeeInputs(
    * so that transaction is source of truth whenever possible.
    */
   useEffect(() => {
-    if (areDappSuggestedAndTxParamGasFeesTheSame(transaction)) {
-      setEstimateUsed('dappSuggested');
-    } else if (transaction?.userFeeLevel) {
+    if (transaction?.userFeeLevel) {
       setEstimateUsed(transaction?.userFeeLevel);
     }
-  }, [setEstimateUsed, transaction]);
+  }, [setEstimateUsed, transaction, userPrefersAdvancedGas]);
```

Comment:
Sorry I'm a bit confused about why `userPrefersAdvancedGas` is required here to updated the estimateUsed? Wouldn't a change in that preference be reflected in a change to the userFeeLevel?